### ISSUE TITLE
Configure Trilateral Datasets in VEDA Dashboard

### ISSUE THREAD

**Issue by aboydnw** [NASA-IMPACT/veda-config#85]
## Description
A bunch of datasets were ingested for the Trilateral dashboard release earlier this year. However, we have yet to configure the VEDA Dashboard to show all of these datasets. This effort is intended to configure the VEDA Dashboard to reveal previously ingested datasets.

Thematic areas may need to be created as well, for datasets that don't already have a thematic area. 
New thematic areas include Agriculture, Cryosphere, Biomass, and Water Quality

Spreadsheet showing datasets ingested for Trilateral dashboard: https://docs.google.com/spreadsheets/d/1WvwifHCN44fntlUKPDBMkfzpvrGQcbyMsBCFtn7Pu1M/edit#gid=0

## Acceptance Criteria
- [ ] Given I am a general public user of the VEDA Dashboard, when I visit the Dashboard home page then I see thematic areas that would contain datasets and discoveries for each thematic area of the Trilateral Dashboard (plus any new ones that VEDA already had configured)
- [ ] Given I am a general public user of the VEDA Dashboard, when I visit thematic area's About page then I can read content that has been curated by NASA to describe this thematic area (outlined in [this folder.](https://drive.google.com/drive/folders/1Li8cEvAdTnN7d2m77uWJ4rG1RLu38whe))
- [ ] Given I am a general public user of the VEDA Dashboard, when I visit each thematic area then I can explore datasets that were ingested for the Trilateral dashboard using the Dataset explorer

**Comment by aboydnw** [comment]
Do we need more information on layers for these datasets?

**Comment by aboydnw** [comment]
@michaelgsuttles I went ahead and linked to the old [Trilateral Dataset List ](https://docs.google.com/spreadsheets/d/1WvwifHCN44fntlUKPDBMkfzpvrGQcbyMsBCFtn7Pu1M/edit#gid=0)in this ticket. Would it make sense to somehow combine that with your spreadsheet tracking datasets moving forward so we have a more complete view of datasets? Same question I suppose would apply to other datasets we have ingested like some COVID datasets for the Air Quality story and all the Environmental Justice datasets.

**Comment by aboydnw** [comment]
1. Need to collaborate with scientists to understand the color map we need
2. Other parameters for visualization
3. Description content
4. Title content
5. Layer names and info content
6. Dataset overview content

Current plan is to put something up on the site and ask Dee-Dee to review with scientists (we can offer to be there if needed) so we can make final tweaks

**Comment by michaelgsuttles** [comment]
Sorry I didn't see this before, @aboydnw. I have mixed feelings about integrating the former datasets into the spreadsheet, but let me look it over and see if it might make sense.

**Comment by aboydnw** [comment]
@michaelgsuttles @danielfdsilva @hanbyul-here remind me, what are our next steps here? do we need to work with any stakeholders/scientists or do we have this information internally?

**Comment by aboydnw** [comment]
@danielfdsilva @hanbyul-here 

@anayeaye and I collaborated on a document here: https://docs.google.com/spreadsheets/d/18VwfBD4h0WWVlZR93tzLhmXJDZH8aDh-O71ZsRMMKTc/edit#gid=773624342

This outlines which collections have yet to be configured in the VEDA Dashboard. The highlighted collections should be configured as part of this ticket. I included links to the dataset view in the Trilateral Dashboard as well as initial thematic areas to drop them into. This can be used as starting points for content.
